src/comments/CommentItem.js

```diff
@@ -60,6 +60,11 @@ export default class CommentItem extends Component {
     }
   }
 
+  updateCommentPostion = () => {
+    // should happen after react-router updated the route
+    setTimeout(() => this.checkHashLink());
```

Comment:
@nil1511 I can access it in `componentWillReceiveProps` but in order to trigger an update from it we should use state which leads to more unnecessary verbosity. I approve the PR for now but I'm afraid it stops working based on future updates of RR or anything similar.